mteb/models/cohere_v.py

```diff
@@ -226,3 +226,25 @@ def get_fused_embeddings(
     use_instructions=False,
     training_datasets=None,
 )
+
+cohere_embed_v4_multimodal = ModelMeta(
+    loader=partial(cohere_v_loader, model_name="embed-v4.0"),
+    name="Cohere/embed-v4.0-multimodal",
+    languages=[],  # Unknown, but supports 100+ languages
```

Comment:
Please add [all languages from the docs](https://docs.cohere.com/docs/cohere-embed?ref=cohere-ai.ghost.io&_gl=1*32bssn*_gcl_au*MzAwMTQyMzE3LjE3NTQ3MzgyNzQ.*_ga*MTk5NzY4Njg4MC4xNzQyNjQyNjAw*_ga_CRGS116RZS*czE3NTQ3MzgyNzQkbzIkZzAkdDE3NTQ3MzgyNzkkajU2JGwwJGgw#list-of-supported-languages).